src/main/java/com/devcycle/sdk/server/local/managers/EnvironmentConfigManager.java

```diff
@@ -129,23 +132,42 @@ private ProjectConfig getConfigResponse(Call<ProjectConfig> call) throws DevCycl
 
         if (response.isSuccessful()) {
             String currentETag = response.headers().get("ETag");
+            String headerLastModified = response.headers().get("Last-Modified");
+
+            if (!this.configLastModified.isEmpty() && headerLastModified != null && !headerLastModified.isEmpty()) {
+                ZonedDateTime parsedLastModified = ZonedDateTime.parse(
+                        headerLastModified,
+                        DateTimeFormatter.RFC_1123_DATE_TIME
+                );
+                ZonedDateTime configLastModified = ZonedDateTime.parse(
+                        this.configLastModified,
+                        DateTimeFormatter.RFC_1123_DATE_TIME
```

Comment:
is this the format we are using everywhere?

Reply:
Yeah - RFC1123 is what cloudflare responds with - thank god it's a standard format.